src/containers/Login/Login.js

```diff
@@ -43,25 +44,40 @@ class Login extends React.Component {
 
     return (
       <form onSubmit={this.handleLoginSubmit}>
-        {!firebaseLoginUserError ? null : <p>{firebaseLoginUserError}</p>}
         {inputsArray.map(([inputName, inputValue], index) => {
           const inputType =
             inputName.toLowerCase() === "password" ? "password" : "text";
 
           return (
-            <input
-              key={index}
-              onChange={this.handleInputChange}
-              type={inputType}
-              value={inputValue}
-              name={inputName}
-              placeholder={inputName}
-              required
-              min="1"
-            />
+            <div className="form-group" key={index}>
+              <label className="c-bundleBlue" htmlFor={inputName}>
+                {inputName}
+              </label>
+              <input
+                className="form-control c-denimBlue bundleBlue-border-bottom-3"
+                onChange={this.handleInputChange}
+                type={inputType}
+                value={inputValue}
+                name={inputName}
+                aria-describedby={`${inputName}`}
+                aria-labelledby={`${inputName}`}
+                aria-label={`${inputName}`}
```

Comment:
Hmm, this is a little improper. The three `aria-` fields have different meanings and you'll never have to use all three. 

* **`aria-label`**: directly the label text that will be read, e.g. `aria-label="Open menu"`.
* **`aria-labelledby`**: unique element ID of a _different_ element that describes what this does. For example, if you have a `button` with an `onClick` whose visualization is an icon, and a text `label` next to it that says `"Open Menu"` and an `id` of `open-menu-label`,  the button would have `aria-labelledby="open-menu-label"`
* **`aria-describedby`**: like `aria-label`, but for long-form descriptions. Instead of `"Open Menu"` it might be a longer form description saying something like `"Button that opens the Help menu"`

In this case, it seems like you could get away with just providing `aria-label` without any of the other ones.